config/c-code/tox.ini.j2

```diff
@@ -1,16 +1,13 @@
 {% set with_coverage = True %}
 {% set raw_testenv_setenv = testenv_setenv %}
-{% set testenv_setenv = ['pure: PURE_PYTHON=1', '!pure-!pypy-!pypy3: PURE_PYTHON=0'] + testenv_setenv %}
+{% set testenv_setenv = ['pure: PURE_PYTHON=1', '!pure-!pypy3: PURE_PYTHON=0'] + testenv_setenv %}
```

Comment:
Are you sure this double negation works? You can test the result without running the actual tests like this: `bin/tox config -k setenv -epy36`

Reply:
I am sure that it will **not** work, but I did not yet have the time ti search for a solution. Currently it seems there is none.